src/base/index.ts

```diff
@@ -24,19 +24,16 @@ export class GrpcUnknownStatus extends Error {
   }
 }
 
-export function grpcResponseToBuffer(
-  response: GrpcResponse
-): Buffer {
-
+export function grpcResponseToBuffer(response: GrpcResponse): Buffer {
   // error messages need to have a zero length message field to be considered valid
-  const message = 'message' in response ? response.message : new Uint8Array();
+  const message = "message" in response ? response.message : new Uint8Array();
```

Comment:
Might be nice to use the same conf as we've got on the frontend, I do prefer single quotes personally :upside_down_face: